### PR TITLE
Bump ledgerjs libs

### PR DESCRIPTION
**Pull request by valpinkman**
Update some core ledgerjs libraries

### Type

Bump

### CODE REVIEW

**Comment by gre** [comment]
that PR ports the LedgerJS new transport impl. there is a bump of node-hid in it , but it should work the same. maybe it will help fixing some usb problem but I highly doubt it. we'll need to see what better can be done than `usb` events

**Comment by valpinkman** [comment]
@Arnaud97234 these issues have been fixes with the latest update of live-common

needs testing tho

File: src/components/ManagerPage/FirmwareUpdate.js [hunk 1]
```diff
@@ -81,18 +81,19 @@ class FirmwareUpdate extends PureComponent<Props, State> {
...
     const { firmware, modal, stepId, ready } = this.state
+
+    const deviceSpecs = getDeviceModel(device.modelId)
+
     return (
       <Card p={4}>
         <Box horizontal align="center" flow={2}>
           <Box color="dark">
-            {device.product === 'Blue' ? <Blue size={30} /> : <NanoS size={30} />}
+            {deviceSpecs.id === 'blue' ? <Blue size={30} /> : <NanoS size={30} />}
```
Comment (gre) [inline src/components/ManagerPage/FirmwareUpdate.js]: note for future refactoring: we should have a component map like this: `{ blue: Blue, nanoS: NanoS }`

Comment (valpinkman) [reply src/components/ManagerPage/FirmwareUpdate.js]: Yup I was thinking about it as well. Will come up with something like this when implementing Nano X designs

**Pull request merged**